src/style.css

```diff
@@ -0,0 +1,148 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.bigSearchBar::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Usually we set font styles to `html` tag, then almost all elements will inherit it (if not, you can set `inherit` yourself)